doc/user-guide.md

```diff
@@ -2,6 +2,8 @@
 
 This guide walks through an example of building a simple memcached-operator using tools and libraries provided by the Operator SDK.
 
+This guide covers the workflow of creating and deploying an operator written in Go. To read about the Ansible Operator, see the [Ansible Operator User Guide][ansible_user_guide]. 
```

Comment:
I would put the word "Ansible" as close as possible to the beginning of the paragraph for people who are visually scanning the document for this section and link. Also as we discussed earlier today, it might be better to avoid calling it the "Ansible Operator". Maybe:

"To learn how to use Ansible to create a memcached operator, see [Ansible Operator User Guide][ansible_user_guide]. The rest of this document will show how to program an operator in Go."